tensorflow_recommenders_addons/dynamic_embedding/python/ops/cuckoo_hashtable_ops.py

```diff
@@ -58,6 +58,7 @@ def __init__(
       name="CuckooHashTable",
       checkpoint=True,
       init_size=0,
+      config={},
```

Comment:
Don't use {}, for different instance will share the same {}, you set default to None and create {} in __init__ function.